consensus/consensus-types/src/quorum_cert.rs

```diff
@@ -43,29 +43,17 @@ impl QuorumCert {
             signed_ledger_info,
         }
     }
-    /// All the vote data getters are just proxies for retrieving the values from the VoteData
-    pub fn certified_block_id(&self) -> HashValue {
-        self.vote_data.proposed().id()
-    }
-
-    pub fn certified_state_id(&self) -> HashValue {
-        self.vote_data.proposed().executed_state_id()
-    }
-
-    pub fn certified_block_round(&self) -> Round {
-        self.vote_data.proposed().round()
-    }
 
-    pub fn parent_block_id(&self) -> HashValue {
-        self.vote_data.parent().id()
+    pub fn vote_data(&self) -> &VoteData {
```

Comment:
do you still need this function?

Reply:
Haha, it is used in one test. I'll land this as is and contemplate that. I've had to rebase this a couple times already...